src/Main.elm

```diff
@@ -188,14 +187,14 @@ update msg model =
             let
                 request : Store.Action
                 request =
-                    Store.CreatePost data
+                    Store.createPost data
```

Comment:
Notice the conversion of `CreatePost` -> `createPost` think of it similarly to how opaque types enable you to control how some inner data is "constructed".